view/adminhtml/templates/info/adyen_hpp.phtml

```diff
@@ -35,27 +35,80 @@
 $_info = $this->getInfo();
 $_isDemoMode = $block->isDemoMode();
 ?>
-<?php if ($_pspReference = $_info->getAdditionalInformation('pspReference')):?>
+<?php if ($_pspReference = $_info->getAdditionalInformation('pspReference')): ?>
     <div>
-        <?php if($_isDemoMode): ?>
-            <?php echo __('Adyen PSP Reference: <a href="https://ca-test.adyen.com/ca/ca/accounts/showTx.shtml?pspReference=%1&txType=Payment" target="__blank">%1</a>', $block->escapeHtml($_pspReference), $block->escapeHtml($_pspReference)) ?>
+        <?php if ($_isDemoMode): ?>
+            <?php echo __('Adyen PSP Reference: <a href="https://ca-test.adyen.com/ca/ca/accounts/showTx.shtml?pspReference=%1&txType=Payment" target="__blank">%1</a>',
```

Comment:
it is `target="_blank"` not `target="__blank"`, right? (two underscores)

Reply:
yes correct. This was already in the code a long time but will change it in the PR.